sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4957,6 +4968,11 @@ pi_result piEnqueueMemBufferCopy(pi_queue Queue, pi_mem SrcBuffer,
   // Copy engine is preferred only for host to device transfer.
   // Device to device transfers run faster on compute engines.
   bool PreferCopyEngine = (SrcBuffer->OnHost || DstBuffer->OnHost);
+
+  // Temporary option added to use copy engine for D2D copy
+  // This is an experimental option and will be removed soon
```

Comment:
Let's not promise it will be "removed soon". By just not documenting it (i.e. not maintaining) we have a freedom to have it forever, if we choose.

Reply:
I removed the comment about "removed soon".